File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:

        heap = []
        seen = set()

        for x in nums:
            converted = -1 * x
            heapq.heappush(heap, converted)


        i = 0
        ans = None

        while i < k:
            ans = -1 * heapq.heappop(heap)
            i+=1

        return ans

        

File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {

        HashMap <Integer, Boolean> seen = new HashMap<>();

        for(int x : nums) {
            
            if(seen.containsKey(x))
                return true;
            
            seen.put(x, true);
        }

        return false;
 
    }
}
